certified-connectors/DeepL/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1799 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "DeepL",
+    "description": "The DeepL API provides programmatic access to DeepL’s machine translation technology, making it possible to bring high quality translation capabilities directly to your websites and applications. If you would like to develop a DeepL plugin for your CAT tool, please contact us.",
```

Comment:
@DeeJayTC please remove the word "API" from the description. please follow these [requirements](https://learn.microsoft.com/en-us/connectors/custom-connectors/certification-submission#write-a-description-for-your-connector)

Reply:
Well, I thought it was okay here as this is our product name. The product is named "DeepL API"

Reply:
I removed the word "The" at the beginning to make clear its a name, is that ok?

Reply:
@Amjed-Ayoub , can we do that? Guess i have to resolve this conversation :)

Reply:
Hi @DeeJayTC, thank you so much for the clarification. Unfortunately, this is one of the very important [requirements](https://learn.microsoft.com/en-us/connectors/custom-connectors/connector-string-guidance#connector-info). 
May suggest since you named the connector "DeepL", in the description instead of saying "The DeepL API ...", can you say "The DeepL connector ....".
Please let me know what you think.

Reply:
Yea changed accordingly @Amjed-Ayoub , changed it to "The DeepL connector"